plugins-compat-tester/src/main/java/org/jenkins/tools/test/hook/WorkflowCpsHooks.java

```diff
@@ -0,0 +1,64 @@
+package org.jenkins.tools.test.hook;
+
+import hudson.model.UpdateSite;
+import hudson.util.VersionNumber;
+import java.util.List;
+import java.util.Map;
+import org.jenkins.tools.test.model.PomData;
+import org.jenkins.tools.test.model.hook.PluginCompatTesterHookBeforeExecution;
+
+public class WorkflowCpsHooks {
+    public static boolean isMultiModuleVersionOfWorkflowCps(Map<String, Object> info) {
+        PomData data = (PomData) info.get("pomData");
+        UpdateSite.Plugin plugin = info.get("plugin") != null ? (UpdateSite.Plugin) info.get("plugin") : null;
+        return isMultiModuleVersionOfWorkflowCps(plugin);
+    }
+
+    public static boolean isMultiModuleVersionOfWorkflowCps(UpdateSite.Plugin plugin) {
+        if (plugin != null && plugin.name.equalsIgnoreCase("workflow-cps") && plugin.version != null) {
+            VersionNumber pluginVersion = new VersionNumber(plugin.version);
+            VersionNumber multiModuleSince = new VersionNumber("2803.v1a_f77ffcc773");
```

Comment:
or just e.g.
```suggestion
            VersionNumber multiModuleSince = new VersionNumber("5000");
```
or whatever it is